### ISSUE TITLE
[crit] Keyboard Navigation: Payments: Focus is lost after activating 'Add Payment Method' button

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5533]
## Action Performed:
Prerequisite: the user us logged in.
1. Using Chrome, open https://staging.new.expensify.com/
2. Navigate to Settings > Payments.
3. Once the Payments overlay opens, press Tab key to focus the 'Add Payment Method' button and press Enter key to activate it.
4. Notice the focus location.
5. Press Tab key and verify if the focus location is visible.

## Expected Result:
The focus should remain on the activated 'Add Payment Method' button.

## Actual Result:
The focus is lost and not visible on the screen. The focus location is not visible when pressing Tab key, thus the keyboard users are not able to continue the flow.

HTML:
`<div tabindex="0" class="css-1dbjc4n r-1p0dtai r-1d2f490 r-orgf3d r-u8s1d r-zchlnj r-ipm5af" style="background-color: rgb(198, 201, 202); height: 629px; width: 1920px;"></div>`

## Workaround:
No workaround

## Area issue was found in:
Add payment method overlay

## Failed WCAG checkpoints
2.4.3

## User impact:
The keyboard users are not able to add any payment method.

## Suggested resolution:
Ensure that the focus remains on the expanded 'Add Payment Method' button. See: https://www.sitepoint.com/learning-to-focus/

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**

https://user-images.githubusercontent.com/88733897/134810486-eb154bb4-ec02-42f1-b45d-0de606d02c91.mp4

![Bug5184111_Focus_order_-_payments](https://user-images.githubusercontent.com/88733897/134810488-83d0012f-3b7a-4f2c-aa3d-b9729068a2b7.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
